src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -2296,49 +2284,51 @@ partial class MyClass
 
 class MyClass2
 {
-}
-        </Document>
-        <Document>
+}";
+            var fixedSource2 = @"
 partial class MyClass
 {
     public int M4() => f2;
 }
 
 static class MyClass3
 {
-}
-        </Document>
-    </Project>
-</Workspace>");
+}";
+
+            await new VerifyCS.Test
+            {
+                TestState = { Sources = { source1, source2 } },
```

Comment:
Original test is validating FixAll in Project. Seems like we are losing that validation in the ported test.

Reply:
Ah my bad, I thought original test was validation across projects. This seems fine.

Reply:
> Original test is validating FixAll in Project.

_All_ code fix tests run all of:

1. Iterative application of individual code fixes
2. Fix All in Document
3. Fix All in Project
4. Fix All in Solution

Any separation of "normal" and "fix all" tests is artificial under the new testing library.